build.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+set -ex
+IMAGE=docker.montagu.dide.ic.ac.uk:5000/vimc-website-builder:i1205
+docker pull $IMAGE
+VOLUME=$1
+BRANCH=$2
+if [ -z $VOLUME ]; then
+    VOLUME_MAP=""
```

Comment:
should this mount the local "static" directory if a volume is not passed in?

Reply:
Did you try `./build.sh .` (see the note in the README)

Reply:
that's what I ran...

Reply:
oh no, sorry, duh!